modin/pandas/test/dataframe/test_default.py

```diff
@@ -762,6 +762,72 @@ def test_resample_specific(rule, closed, label, on, level):
         )
 
 
+@pytest.mark.parametrize(
+    "params",
+    [
+        ["volume"],
+        ("volume",),
+        pandas.Series(["volume"]),
+        pandas.Index(["volume"]),
+        ["volume", "price", "date"],
+    ],
+    ids=["list", "tuple", "series", "index", "error_params"],
+)
+def test_resample_getitem(params):
+    idx = pandas.DatetimeIndex(
+        [
+            "2013-07-31",
+            "2013-08-31",
+            "2013-09-30",
+            "2013-10-31",
+            "2013-11-30",
+            "2013-12-31",
+            "2014-01-31",
+            "2014-02-28",
+            "2014-03-31",
+            "2014-04-30",
+            "2014-05-31",
+            "2014-06-30",
+        ],
+        dtype="datetime64[ns]",
+        freq="M",
+    )
+
+    md_idx = pd.DatetimeIndex(idx)
```

Comment:
```suggestion
    index = pandas.date_range("1/1/2013", periods=9, freq='T')
```

Your data range can't be downsampled with by `3T`. In the result you generate too large dataframes as a result of Resampelr operations.